keep/common/fedora.py

```diff
@@ -319,8 +319,14 @@ def save(self, logMessage=None):
 
                 raise DuplicateContent(msg, pids, pid_cmodels)
 
-        return super(DigitalObject, self).save(logMessage)
+        # check if the mods content title of the object has been changed
+        # if changed, apply the change to the object in pidman
+        pidman_label = self.pidman_client.get_ark(self.noid)['name']
```

Comment:
This pid api call should only happen if the mods has been modified - we don't want to make an api call if we don't need to.